addons/mail/static/src/core/web/follower_list_dropdown.js

```diff
@@ -0,0 +1,117 @@
+import { Component, useState, onWillUnmount, onMounted } from "@odoo/owl";
+import { _t } from "@web/core/l10n/translation";
+import { useService } from "@web/core/utils/hooks";
+import { FollowerSubtypeDialog } from "./follower_subtype_dialog";
+import { DropdownItem } from "@web/core/dropdown/dropdown_item";
+import { Dropdown } from "@web/core/dropdown/dropdown";
+import { useDropdownState } from "@web/core/dropdown/dropdown_hooks";
+import { FollowerList } from "@mail/core/web/follower_list";
+
+/**
+ * @typedef {Object} Props
+ * @property {function} [onAddFollowers]
+ * @property {function} [onFollowerChanged]
+ * @property {import('@mail/core/common/thread_model').Thread} thread
+ * @extends {Component<Props, Env>}
+ */
+
+let local_id = 0;
+
+export class FollowerListDropDown extends Component {
+    static template = "mail.FollowerListDropDown";
+    static components = { DropdownItem, Dropdown, FollowerList };
+    static props = ["onAddFollowers?", "onFollowerChanged?", "thread"];
+
+    setup() {
+        super.setup();
+        this.action = useService("action");
+        this.store = useState(useService("mail.store"));
+        this.followerListId = local_id++;
+        this.followerListDropdown = useDropdownState({
+            onOpen: async () => {
+                await this.followerListView.loadFollowers(0);
```

Comment:
IMO there is no need to `await` here. Unless it helps in removing some UI flickers.